src/backend/base/langflow/custom/custom_component/component.py

```diff
@@ -326,8 +326,7 @@ def _set_output_required_inputs(self) -> None:
                 source_code = inspect.getsource(method)
                 ast_tree = ast.parse(dedent(source_code))
             except Exception:  # noqa: BLE001
-                source_code = self._code
-                ast_tree = ast.parse(dedent(source_code))
+                ast_tree = ast.parse(dedent(self._code or ""))
```

Comment:
`self._code` is not None by construction

Reply:
LGTM